hello_world/hello_transaction.js

```diff
@@ -6,11 +6,12 @@ const {
 class HelloTransaction extends BaseTransaction {
 
 	static get TYPE () {
-		return 10;
+		return 20;
 	};
 
 	static get FEE () {
-		return `${10 ** 8}`;
+		//return `${10 ** 8}`;
```

Comment:
Fee set to zero, shall we leave it like that?

Reply:
@michielmulders there is something strange with the genesis accounts... I noticed in the tutorial I was not able to send the hello transaction because of missing funds.

There seems to be a new address for the genesis account. I am investigating this atm.

Reply:
I updated the genesis account now 👍